src/Core/ManifestProcessor.php

```diff
@@ -22,6 +22,9 @@ public function __construct(iterable $makers)
 
     public function process(ManifestInterface $manifest, AppInterface $app): void
     {
+        if($manifest instanceof AppAwareInterface) {
+            $manifest->setApp($app);
```

Comment:
I think we should move this logic to bundle, and set app to manifests in compiler pass.
That way, manifests will have app to work with. 
Otherwise - users will see property $app, but it will be always empty.